electronics_lib/JLC_Resistor.py

```diff
@@ -0,0 +1,87 @@
+class JLCPCB_ResistorTable(ESeriesResistor):
+  RESISTANCE = PartsTableColumn(Range)  
+  POWER_DISSIPATION = PartsTableColumn(Range)  
+  PRICE = PartsTableColumn(float)
```

Comment:
Consider creating a `JlcPcbTable` class that contains common definitions for all JLCPCB tables, like the `DigikeyTable` class in #66. Here, this would just be the price.